src/pybind/mgr/dashboard/frontend/src/app/ceph/dashboard-v3/dashboard/dashboard-v3.component.html

```diff
@@ -216,54 +216,48 @@
-          <cd-card cardTitle="Cluster Utilization"
-                   i18n-title
-                   aria-label="Cluster utilization card">
-            <div class="ms-4 me-4 mt-0">
-              <cd-dashboard-time-selector (selectedTime)="getPrometheusData($event)">
-              </cd-dashboard-time-selector>
-              <ng-container *ngIf="capacity">
-                <cd-dashboard-area-chart chartTitle="Used Capacity (RAW)"
-                                         [maxValue]="capacity.total_bytes"
-                                         dataUnits="B"
-                                         label="Used Capacity"
-                                         [data]="queriesResults.USEDCAPACITY">
-                </cd-dashboard-area-chart>
-              </ng-container>
-              <cd-dashboard-area-chart chartTitle="IOPS"
-                                       dataUnits=""
-                                       decimals="0"
-                                       label="Reads"
-                                       label2="Writes"
-                                       [data]="queriesResults.READIOPS"
-                                       [data2]="queriesResults.WRITEIOPS">
-              </cd-dashboard-area-chart>
-              <cd-dashboard-area-chart chartTitle="OSD Latencies"
-                                       dataUnits="ms"
-                                       decimals="2"
-                                       label="Apply"
-                                       label2="Commit"
-                                       [data]="queriesResults.READLATENCY"
-                                       [data2]="queriesResults.WRITELATENCY">
-              </cd-dashboard-area-chart>
-              <cd-dashboard-area-chart chartTitle="Client Throughput"
-                                       dataUnits="B/s"
-                                       decimals="2"
-                                       label="Reads"
-                                       label2="Writes"
-                                       [data]="queriesResults.READCLIENTTHROUGHPUT"
-                                       [data2]="queriesResults.WRITECLIENTTHROUGHPUT">
+      <div class="col-sm-12 d-flex flex-column pt-4">
+        <cd-card cardTitle="Cluster Utilization"
+                 i18n-title
+                 aria-label="Cluster utilization card">
+          <div class="ms-4 me-4 mt-0">
+            <cd-dashboard-time-selector (selectedTime)="getPrometheusData($event)">
+            </cd-dashboard-time-selector>
+            <ng-container *ngIf="capacity">
+              <cd-dashboard-area-chart chartTitle="Used Capacity (RAW)"
+                                       [maxValue]="capacity.total_bytes"
+                                       dataUnits="B"
+                                       [labelsArray]="['Used Capacity']"
+                                       [dataArray]="[queriesResults.USEDCAPACITY]">
```

Comment:
instead of calling it `labelsArray` and `dataArray` we can just leave them `labels` and `data`. And if the input is properly typed to recieve array, I guess that would be okay. If you want you can leave some comment in the other component where we initialize `labelsArray` to tell what exactly is the expectation of the input.
 
```suggestion
                                       [labels]="['Used Capacity']"
                                       [data]="[queriesResults.USEDCAPACITY]">
```

Reply:
okay. thanks. can you remove the `pe-4` in line 7 of this file. Its an unnecessary space and I don't want to create a separate PR just for it?

Reply:
Sure, Done